dash/dash-renderer/src/components/error/FrontEnd/FrontEndErrorContainer.react.js

```diff
@@ -28,11 +24,8 @@ class FrontEndErrorContainer extends Component {
             <div className={cardClasses}>
                 <div className='dash-error-card__topbar'>
                     <div className='dash-error-card__message'>
-                        🛑 Errors (
-                        <strong className='test-devtools-error-count'>
```

Comment:
This className is required for the tests and need to be put back or the tests adapted.

Reply:
Oh ok I can add that back

Reply:
Ok ended up just adding the class to where the new error count is. Waiting on the CI now